src/client/Components/CreateCollection/CreateCollection.js

```diff
@@ -0,0 +1,38 @@
+import styles from './CreateCollection.module.css';
```

Comment:
This will need to be updated to fit with the consistency of the rest of the files.